cpp/src/arrow/ipc/json_integration_test.cc

```diff
@@ -272,7 +272,7 @@ static const char* JSON_EXAMPLE = R"example(
         {
           "name": "foo",
           "count": 5,
-          "DATA": [1, 2, 3, 4, 5],
+          "DATA": ["1", "2", "3", "4", "5"],
```

Comment:
add literal for INT64_MIN and INT64_MAX.

Reply:
Will do